CMSgov/mint-app
# Easi 3368/ update cms component

# EASI-3368

## Changes and Description

- BE
   - Remove the `OTHER` option from CMS_Centers
   - Drop the other option text field
   - Update unit tests
- FE
    - Update translations for CMS_Center
    - Remove Other text field
    - Updated basics schema
    - Snapshot updates



## How to test this change
1. Bring up the app
2. Make sure you can no longer select other for CMS Center
3. Make sure that there is no longer an other text field
4.  Make sure all unit tests pass.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
